recipes/drfp/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "drfp" %}
+{% set version = "0.2.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/drfp-{{ version }}.tar.gz
+  sha256: ea4755ad989bbd31a62cc628a7ae1bacdf2bfef0dfde5c899cb4551917405640
+
+build:
+  number: 0
+  entry_points:
+    - drfp = drfp.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - click >=8.0.1
+    - importlib_metadata  # [py<38]
+    - numpy >=1.21.0
+    - python
```

To finish making this noarch, add Python version constraints and remove the selector on `importlib_metadata` (better to have this noarch and unconditionally install `importlib_metadata`):
```suggestion
  host:
    - pip
    - python >=3.7
  run:
    - click >=8.0.1
    - importlib_metadata
    - numpy >=1.21.0
    - python >=3.7
```